panther_analysis_tool/main.py

```diff
@@ -1020,8 +1045,11 @@ def classify_analysis(
             # extra validation for simple detections based on json schema
             if is_simple_detection(analysis_spec):
                 jsonschema.validate(analysis_spec, SIMPLE_DETECTION_SCHEMA)
+                classified_specs[SIMPLE_DETECTION].append(
+                    (analysis_spec_filename, dir_name, analysis_spec)
+                )
```

Comment:
What is this doing / what does it get used by?

Reply:
This is used [here](https://github.com/panther-labs/panther_analysis_tool/pull/285/files#diff-4c3406e46ca13890fa64d295f86ed01e0b0bcd3d28031f1fc97978bb8fca2da3R730), so that we don't have to iterate through the detection list again in order to batch up the request to the backend to transpile.